integration/test_redis_specific.py

```
@@ -25,46 +27,69 @@ def redis():
+    except:
+            pass
+    finally:
+        for key in driver.redis.scan_iter(f'*storey:{table_name}*'):
+            driver.redis.delete(key)
 
 def test_redis_driver_join(redis):
-    driver = RedisDriver(redis)
-    table = Table('test', driver)
+    try:
+        table_name = 'test_redis_driver_join'
 
-    # Create the data we'll join with in Redis.
-    driver.redis.hset("storey:test/2:static", mapping={"name": "1234"})
-    controller = build_flow([
-        SyncEmitSource(),
-        JoinWithTable(table, lambda x: x['col2']),
-        Reduce([], lambda acc, x: append_return(acc, x))
-    ]).run()
+        driver = RedisDriver(redis)
+        table = Table(table_name, driver)
+        table_name = f'{table_name}/'
 
-    controller.emit({'col1': 1, 'col2': '2'}, 'key')
-    controller.emit({'col1': 1, 'col2': '2'}, 'key')
-    controller.terminate()
-    termination_result = controller.await_termination()
+        # Create the data we'll join with in Redis.
+        hash_key = RedisDriver.make_key('storey:', table_name, '2')
+        redis_key = RedisDriver._static_data_key(hash_key)
 
-    expected_result = [
-        {'col1': 1, 'col2': '2', 'name': 1234},
-        {'col1': 1, 'col2': '2', 'name': 1234}]
+        driver.redis.hset(redis_key, mapping={"name": "1234"})
+        controller = build_flow([
+            SyncEmitSource(),
+            JoinWithTable(table, lambda x: x['col2']),
+            Reduce([], lambda acc, x: append_return(acc, x))
+        ]).run()
 
-    assert termination_result == expected_result
+        controller.emit({'col1': 1, 'col2': '2'}, 'key')
+        controller.emit({'col1': 1, 'col2': '2'}, 'key')
+        controller.terminate()
+        termination_result = controller.await_termination()
+
+        expected_result = [
+            {'col1': 1, 'col2': '2', 'name': 1234},
+            {'col1': 1, 'col2': '2', 'name': 1234}]
+
+        assert termination_result == expected_result
+    except:
+        pass
```

Why are we ignoring all errors?

---

right...should be 'raise' :)